src/Tribe/Editor/Meta.php

```diff
@@ -99,6 +101,61 @@ public function register() {
 		register_meta( 'post', '_VenueLng', $this->text() );
 	}
 
+	/**
+	 * Short-circuits deleting metadata items that dont exist, for compatibility purposes we need to make sure
+	 * WordPress doesn't throw an error when the meta is not present.
+	 *
+	 * @since TBD
+	 *
+	 * @param null|bool $delete     Whether to allow metadata deletion of the given type.
+	 * @param int       $object_id  ID of the object metadata is for.
+	 * @param string    $meta_key   Metadata key.
+	 * @param mixed     $meta_value Metadata value. Must be serializable if non-scalar.
+	 * @param bool      $delete_all Whether to delete the matching metadata entries
+	 *                              for all objects, ignoring the specified $object_id.
+	 *                              Default false.
+	 *
+	 * @return bool
+	 */
+	public function filter_allow_meta_delete_non_existent_key( $delete, $object_id, $meta_key, $meta_value, $delete_all ) {
+		if ( ! function_exists( 'wp_is_json_request' ) || ! wp_is_json_request() ) {
+			return $delete;
+		}
+
+		$meta_keys_to_allow = [
+			'_EventOrganizerID' => true,
+			'_EventVenueID' => true,
+		];
+
+		if ( ! isset( $meta_keys_to_allow[ $meta_key ] ) ) {
```

Comment:
Question/thought: will `'_EventVenueID' => false,` trigger this? Should it?

Reply:
It should, this method is just faster then `in_array`.